### PR TITLE
update to 0.22.0

### PR DESCRIPTION
**Pull request by cbouss**
Changes:
- update to 0.22.0

https://github.com/SciTools/cartopy/tree/v0.22.0

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
   name: cartopy
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/C/Cartopy/Cartopy-{{ version }}.tar.gz
-  sha256: 89d5649712c8582231c6e11825a04c85f6f0cee94dbb89e4db23eabca1cc250a
+  sha256: b300f90120931d43f11ef87c064ea1dacec1b59a4940aa76ebf82cf09548bb49
 
 build:
   # s390x is missing geos and this package is not in the SoW
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml]: since geos is no longer used, you could change this comment

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM, but the comment

File: recipe/meta.yaml (hunk 1 above)
Comment (cbouss) [reply recipe/meta.yaml]: Addressed, ty

File: recipe/meta.yaml [hunk 2]
```diff
@@ -23,39 +23,31 @@ requirements:
...
     - pyshp >=2.1
-    - geos >=3.7.2
-    # including matplotlib-base even though it's optional upstream
-    - matplotlib-base >=3.1
-    - pyproj >=3.0.0
+    - matplotlib-base >=3.4
+    - pyproj >=3.1.0
   run_constrained:
     - scipy >=1.3.1
+    - pillow >=6.1.0
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:38]:
```suggestion
    - pillow >=6.1.0
    - OWSLib>=0.20.0
```
I don't think we have this package, but it doesn't hurt to add it in the `run_constained`. https://github.com/SciTools/cartopy/blob/v0.22.0/pyproject.toml#L54C9-L54C23

**Review by JeanChristopheMorinPerso — APPROVED** [review]

File: recipe/meta.yaml (hunk 2 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml:38]: We do have `owslib 0.24.1` (the latest version)

File: recipe/meta.yaml [hunk 3]
```diff
@@ -23,39 +23,31 @@ requirements:
...
-    # Expected:
-    #   {config[data_dir]}/shapefiles/gshhs/{scale}/GSHHS_{scale}_L{level}.shp
-    # Got:
-    #   {config[data_dir]}\shapefiles\gshhs\{scale}\GSHHS_{scale}_L{level}.shp
-    - python -m pytest --pyargs --doctest-modules cartopy || cmd /K "exit /b 0"  # [win]
+    - pytest -ra -n 4 --pyargs cartopy
+
 about:
   home: https://scitools.org.uk/cartopy
   license: LGPL-3.0-or-later
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: The folder `lib/cartopy/data` from the [source dist](https://pypi.org/project/Cartopy/0.22.0/#files) contains a license [file](https://github.com/SciTools/cartopy/blob/v0.22.0/lib/cartopy/data/LICENSE) for data files. Three of them are Public Domain and one is LGPL. So we should add this license file to `license_file` and modify the license as `LGPL-3.0-or-later AND  OGL-UK-1.0 OR OGL-UK-2.0`.
See:
- https://scitools.org.uk/cartopy/docs/latest/copyright.html#cartopy-documentation-and-examples
- https://www.metoffice.gov.uk/hadobs/hadisst/terms_and_conditions.html
- https://www.naturalearthdata.com/about/terms-of-use/
- https://www.earthdata.nasa.gov/learn/use-data/data-use-policy

I'm not sure on why the upstream source contains GPL3 license file https://github.com/SciTools/cartopy/blob/main/COPYING

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**